docs/guides/integrating-cert-manager.md

```diff
@@ -0,0 +1,290 @@
+
+## Details
+
+### Step 1 – Deploy cert-manager
+
+The first step is to deploy cert-manager onto the cluster.
+
+1. Add the Helm repository
+
+```shell
+    helm repo add jetstack https://charts.jetstack.io
+	helm repo update
+```
+
+1. Install cert-manager, and enable the GatewayAPI feature gate:
+
+```shell
+helm install \
+  cert-manager jetstack/cert-manager \
+  --namespace cert-manager \
+  --create-namespace \
+  --version v1.12.0 \
+  --set installCRDs=true \
+  --set "extraArgs={--feature-gates=ExperimentalGatewayAPISupport=true}"
+```
+
+### Step 2 – Create a ClusterIssuer
+
+Next we need to create a ClusterIssuer. Issuers, and ClusterIssuers, are Kubernetes resources that represent certificate
+authorities (CAs) that are able to generate signed certificates by honouring certificate signing requests. The only
+difference between Issuers and ClusterIssuers is that Issuers are namespaced resources, and ClusterIssuers are not.
+
+We are using the ACME Issuer type, and Let's Encrypt as the CA server. In order for Let's Encypt to verify that we own
+the domain a certificate is being requested for, we must complete "challenges". This is to ensure clients are
+unable to request certificates for domains they do not own. We will configure the Issuer to use a HTTP01 challenge, and
+our Gateway resource that we will create in the next step as the solver. To read more about HTTP01 challenges, see
+[here](https://cert-manager.io/docs/configuration/acme/http01/). Use the following YAML definition to create the
+resource, but please note the `email` field should be updated to your own email address.
+
+```yaml
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-prod
+spec:
+  acme:
+    # You must replace this email address with your own.
+    # Let's Encrypt will use this to contact you about expiring
+    # certificates, and issues related to your account.
+    email: my-name@example.com
+    server: https://acme-v02.api.letsencrypt.org/directory
+    privateKeySecretRef:
+      # Secret resource that will be used to store the account's private key.
+      name: issuer-account-key
+    # Add a single challenge solver, HTTP01 using NKG
+    solvers:
+    - http01:
+        gatewayHTTPRoute:
+          parentRefs: # This is the name of the Gateway that will be created in the next step
+            - name: gateway
+              namespace: default
+              kind: Gateway
+```
+
+### Step 3 – Deploy our Gateway with the cert-manager annotation
+
+Next we need to deploy our Gateway. Use can use the below YAML manifest, updating the `spec.linsteners[0].hostname`
+field to the required value for your environment.
+
+```yaml
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: Gateway
+metadata:
+  name: gateway
+  annotations: # This is the name of the ClusterIssuer created in the previous step
+    cert-manager.io/cluster-issuer: letsencrypt-prod
+spec:
+  gatewayClassName: nginx
+  listeners:
+  - name: http
```

Comment:
because we don't use HTTP protocol at all  (the HTTPRoute doesn't reference it) - let's remove this listener to reduce complexity

Reply:
We need it for the temp HTTPRoute though, as that requires port 80